README.md

```diff
@@ -7,7 +7,7 @@ The driving use case of SOMA is for single-cell data in the form of annotated ma
 
 ## Motivation
 
-Datasets generated by profiling single cells are rapidly increasing in size and complexity. This has resulted in a need for scalable solutions to accommodate data sizes that no longer fit in memory and flexibility to accommodate the diversity of data being produced. 
```

Comment:
I ran `alias pav='python -m pre_commit run -a -v'` on my laptop & got this along for the ride :(

Reply:
pre-commit does not care about unchanged files; you can revert this file.